app/src/main/java/de/westnordost/streetcomplete/quests/foot/ProhibitedForPedestriansAnswer.kt

```diff
@@ -3,6 +3,5 @@ package de.westnordost.streetcomplete.quests.foot
 enum class ProhibitedForPedestriansAnswer {
     YES,
     NO,
-    IS_LIVING_STREET,
```

Comment:
I guess this was removed on purpose? Now a test is failing, see https://github.com/streetcomplete/StreetComplete/actions/runs/8960282103/job/24606606961:

> app/src/test/java/de/westnordost/streetcomplete/quests/foot/AddProhibitedForPedestriansTest.kt:57:39 Unresolved reference: IS_LIVING_STREET

Reply:
oops. Thanks!

Reply:
Fixed in https://github.com/streetcomplete/StreetComplete/commit/1fe1f79925c1b72592212f64d5b31609a03ac288